Makefile

```diff
@@ -9,18 +9,32 @@ IMAGE_NAME := node-feature-discovery
 IMAGE_TAG_NAME := $(VERSION)
 IMAGE_REPO := $(IMAGE_REGISTRY)/$(IMAGE_NAME)
 IMAGE_TAG := $(IMAGE_REPO):$(IMAGE_TAG_NAME)
-
+K8S_NAMESPACE := kube-system
 
 all: image
 
-image:
+image: nfd-master.yaml nfd-worker-daemonset.yaml
```

Comment:
We have other yaml, too, namely `nfd-worker-job.yaml.template` and `nfd-daemonset-combined.yaml.template`

Reply:
Good point. Added `yaml_templates` variable to Makefile to find them all.